ads/navegg.md

```diff
@@ -0,0 +1,45 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Navegg
+
+If you want to serve ads with Navegg data you need to use this adapter.
+
+The Navegg adapter only supports Doubleclick for now.
+
+## Example
+
+```html
+<amp-ad width=320 height=50
+  type="navegg"
+  data-acc="XXXXX"
+  data-slot="/4119129/mobile_ad_banner"
```

Comment:
We intend to be an extension to the doubleclick Ads, all of these additional parameters coming from the doubleclick documentation. We append a line in the md file explaining that.